bundlewrap/items/actions.py

```diff
@@ -69,6 +70,8 @@ def _get_result(
                 return (self.STATUS_SKIPPED, ["unless"])
 
         question_body = self.attributes['command']
+        if self.attributes['data_stdin'] is not None:
+            question_body += "\n\n<data being fed to stdin>"
```

Comment:
Missing _() for translation. Also I think "(data is being fed to stdin)" would be slightly more clear since angle brackets usually indicated some form of placeholder. Or just integrate it into the command display:
```
<data> | my command
```